tests/on-device/lib.bash

```diff
@@ -20,6 +20,9 @@ setup() {
 
     ON_DEVICE_TESTS_SETUP=1
   fi
+
+  # restore an generation before start testing
+  $DEFAULT_ACTIVATE_SCRIPT
```

Comment:
Not exactly. If you are developing on the nix-on-droid script, this line will revert everything and go back to the initial state. switch_to_default_config will then activate the latest version of nix-on-droid you want to test and only then you can start testing.

Reply:
OK. I propose `restore to initial generation before the start of each test` or `restore to pre-testing generation before the start of each test` then.

Reply:
Not sure if you saw my rewording `restore an generation of a stable release before start testing` already. Is that fine? Otherwise I'll  pick one of yours.

Reply:
yeah, I was commenting on the new one from the commit view. `a`/`and` + `start`/`starting`  + unclear what `stable release` means when the reverting point is fully arbitrary now as you've switched it, philosophically, from testing default <-> different confs to "state under test" <-> different confs. Not that there's anything wrong with it.

Reply:
Agree, I like your second wording more than mine, will change it :)